writer.go

```diff
@@ -744,21 +718,6 @@ func (p *MediaPlaylist) Close() {
 	p.Closed = true
 }
 
-// SetDefaultKey sets encryption key appeared once in header of the
```

Comment:
Why have you removed the SetDefaultKey method?
It may not be used right now, but since this is an external library, we should do as few changes as possible.

Reply:
I **was** thinking `SetDefaultKey()` was unused, but it is a public method so you are right ofc.